config/config.go

```diff
@@ -36,10 +37,12 @@ type AllowedFailures struct {
 	RecentLogs            int `json:"recent_logs"`
 	StreamingLogs         int `json:"streaming_logs"`
 	AppSyslogAvailability int `json:"app_syslog_availability"`
+	TCPAvailability       int `json:"tcp_availability"`
```

Comment:
Is this used anywhere?

Reply:
Aah! Nice catch

Reply:
It's used in creating the Measurement object:
```
return measurement.NewPeriodic(
    logger,
    clock,
    time.Second,
    tcpAvailabilityMeasurement,
    measurement.NewResultSet(),
    allowedFailures.TCPAvailability,
    func(string, string) bool { return false },
)
```